docs/source/tutorials/staging_data.ipynb

```diff
@@ -1,20 +1,340 @@
 {
```

Comment:
<p>There is no need in a notebook to break the markdown across multiple cells. N consecutive markdown cells can be merged, until a code cell is needed. That makes editing the notebook easier and maybe makes the interpretation a bit faster but not I do not have data about that.</p><p> </p><p>One necessary thing to do is using consistent headings: '#' for the title, '##' for section, '###' for subsection and so on. That is necessary because <code>sphinx</code> build the TOC of the tutorials with a specified depth that is calculated based on the number of '#' in the headings. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2882/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>yeah, that is considered</p>

Reply:
<p>Prior knowledge (e.g., pilot, task, sandbox) is required to understand Staging Data. Maybe we can start with the previous tutorials we assume the user should read before this.&nbsp;</p>